src/components/popup-menu/popup-menu.scss

```diff
@@ -0,0 +1,38 @@
+$userPanelBackground: $blueSecondary;
+$userPanelPadding: $layoutDefaultPadding;
+
+$include-html: false !default;
+
+@if ($include-html) {
+  .mint-popup-menu {
+    position: relative;
+    width: 100%;
+    padding: $userPanelPadding/3 $userPanelPadding/2;
```

Comment:
Hm, for simplicity I would use variables only when they might be needed outside, or used in more than one place in file.